reference/docs-conceptual/getting-started/cookbooks/Decode-PowerShell-Command-from-a-Running-Process.md

```diff
@@ -0,0 +1,92 @@
+# Decode PowerShell Command from a Running Process
+
+There are times when I've found a PowerShell process running that is taking up a bunch of resources. Sometimes they've even been my own scripts running in the context of a [Task Scheduler](https://docs.microsoft.com/windows/desktop/TaskSchd/task-scheduler-start-page
```

Comment:
Change this to second person. See the [Microsoft Style Guide](https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=25515). Try something like:


> At some time you may have a PowerShell process that is using a large amount of system resources. This process
> could be running in the context of a Scheduled Task or SQL Server Agent job.

Reply:
Done.